runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -187,3 +187,29 @@ J9::ARM64::TreeEvaluator::monentEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return targetRegister;
    }
 
+TR::Register *
+J9::ARM64::TreeEvaluator::arraylengthEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_J9VMBase *fej9 = (TR_J9VMBase *)(cg->fe());
+   // ldrw R, [B + contiguousSize]        
+   // cmp R, 0                            
+   // cselw R, [B + discontiguousSize]    
```

Comment:
Are these the `AArch64` instructions/pseudo-code for the operation? If not are the comments necessary?

Reply:
Yes I wrote those for AArch64.